include/unifex/task.hpp

```diff
@@ -165,6 +313,8 @@ struct _task<T>::type {
 };
 } // namespace _task
 
+using _task::with_awaitable_senders;
```

Comment:
I'm not sure we want this in the public interface just yet.

It's design will need to change once we start forwarding through context from the coroutine to awaited sender.